Classes/Service/ContentService.php

```diff
@@ -225,8 +226,6 @@ public function moveRecord(array &$row, &$relativeTo, $parameters, DataHandler $
 			} elseif (0 < (integer) $relativeTo) {
 				// moving to first position in colPos, means that $relativeTo is the pid of the containing page
 				$row['sorting'] = $tceMain->getSortNumber('tt_content', 0, $relativeTo);
-				$row['tx_flux_parent'] = NULL;
```

Comment:
These must not be removed; they were added as part of a recent bugfix. Perhaps you need a rebase?